From: Christopher Hoskin <mans0954@debian.org>
Date: Thu, 29 Dec 2022 15:07:09 +0000
Subject: configure

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index df48cb9..2370a75 100755
--- a/configure
+++ b/configure
@@ -3515,7 +3515,7 @@ else
     echo "$ac_t""yes" 1>&6
 fi
 
-for ac_prog in $PYTHON python
+for ac_prog in $PYTHON python2
 do
 # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
