From: Christopher Hoskin <mans0954@debian.org>
Date: Thu, 29 Dec 2022 15:07:10 +0000
Subject: funnelweb

---
 filters/ptprince/Makefile.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/filters/ptprince/Makefile.in b/filters/ptprince/Makefile.in
index 7b6ce7f..dc7f7c2 100644
--- a/filters/ptprince/Makefile.in
+++ b/filters/ptprince/Makefile.in
@@ -106,7 +106,7 @@ DBGFLAGS += -Zi
 endif
 
 ifeq ($(OS_ARCH),Linux)
-FW=funnelweb
+FW=/usr/bin/fw
 DBGFLAGS += -g
 endif
 
@@ -117,7 +117,7 @@ endif
 
 
 $(CPPSRCS):$(FWSRCS)
-	-$(srcdir)/$(FW) +t +D tex
+	-$(FW) +t +D tex
 
 
 # why doesn't this work?
