--- a/configure.in
+++ b/configure.in
@@ -4150,8 +4150,8 @@
 MOZ_ENABLE_CANVAS=1
 MOZ_EXTENSIONS_ALL=" wallet xml-rpc venkman inspector irc typeaheadfind gnomevfs sroaming datetime finger cview layout-debug tasks sql xforms schema-validation reporter"
 MOZ_FEEDS=1
-MOZ_IMG_DECODERS_DEFAULT="png gif jpeg bmp xbm icon"
-MOZ_IMG_ENCODERS_DEFAULT="png jpeg"
+MOZ_IMG_DECODERS_DEFAULT="png gif bmp xbm icon"
+MOZ_IMG_ENCODERS_DEFAULT="png"
 MOZ_IPCD=
 MOZ_JAVAXPCOM=
 MOZ_JSDEBUGGER=1
