From: Christopher Hoskin <mans0954@debian.org>
Date: Thu, 29 Dec 2022 15:07:10 +0000
Subject: mkshlib

---
 security/coreconf/Linux.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/security/coreconf/Linux.mk b/security/coreconf/Linux.mk
index 765fe21..4c20f2a 100644
--- a/security/coreconf/Linux.mk
+++ b/security/coreconf/Linux.mk
@@ -147,6 +147,8 @@ ifeq ($(OS_RELEASE),2.0)
          sed -e 's,;+,,' -e 's; DATA ;;' -e 's,;;,,' -e 's,;.*,;,' > $@
 endif
 
+MKSHLIB         = $(CC) -shared -fpermissive -std=c++03 -Wl,-soname -Wl,$(@:$(OBJDIR)/%.so=%.so)
+
 ifdef BUILD_OPT
 ifeq (11,$(ALLOW_OPT_CODE_SIZE)$(OPT_CODE_SIZE))
 	OPTIMIZER = -Os
