--- /dev/null
+++ b/mozconfigs/linux/Prince.mozconfig
@@ -0,0 +1,16 @@
+# sh
+# Build configuration script
+#
+# See http://www.mozilla.org/build/unix.html for build instructions.
+#
+# ac_add_options --disable-static --disable-libxul
+#
+# Options for client.mk.
+
+. $topsrcdir/prince/config/mozconfig
+
+mk_add_options MOZ_OBJDIR=@TOPSRCDIR@/../debug
+mk_add_options MOZ_MAKE_FLAGS=" -j4"
+ac_add_options --enable-debug							
+ac_add_options --disable-optimize
+ac_add_options --enable-chrome-format=flat
--- /dev/null
+++ b/mozconfigs/linux/PrinceRelease.mozconfig
@@ -0,0 +1,21 @@
+# sh
+# Build configuration script
+#
+# See http://www.mozilla.org/build/unix.html for build instructions.
+#
+# ac_add_options --disable-static --disable-libxul
+#
+# Options for client.mk.
+
+
+
+. $topsrcdir/prince/config/mozconfig
+
+CPPFLAGS="-Wno-error=narrowing -I/usr/include/freetype2/"
+HOST_CPPFLAGS="-Wno-error=narrowing -I/usr/include/freetype2/"
+CXXFLAGS="-fpermissive"
+
+ac_add_options --with-system-nss
+
+mk_add_options MOZ_OBJDIR=@TOPSRCDIR@/../release
+mk_add_options MOZ_MAKE_FLAGS=" -j4"
