--- a/gfx/src/thebes/nsThebesDeviceContext.cpp
+++ b/gfx/src/thebes/nsThebesDeviceContext.cpp
@@ -57,7 +57,6 @@
 #include "nsFont.h"
 
 #include <pango/pango.h>
-#include <pango/pangox.h>
 #include <pango/pango-fontmap.h>
 #endif /* GTK2 */
 
--- a/gfx/src/thebes/nsSystemFontsGTK2.cpp
+++ b/gfx/src/thebes/nsSystemFontsGTK2.cpp
@@ -48,7 +48,6 @@
 #include <gdk/gdkx.h>
 
 #include <pango/pango.h>
-#include <pango/pangox.h>
 #include <pango/pango-fontmap.h>
 
 #include <fontconfig/fontconfig.h>
